core/src/main/scala/com/evolutiongaming/kafka/flow/KeyFlow.scala

```diff
@@ -17,72 +20,84 @@ object KeyFlow {
 
   /** Create flow which persists snapshots, events and restores state if needed */
   def of[F[_]: Monad: Ref.Make: KeyContext, S, A](
+    key: KafkaKey,
```

Comment:
a bunch of broken apis, will you bump the major version? you could keep the original methods as deprecated and call the new ones with a fake instance of `KafkaKey`

Reply:
Yes, I planned to increase the major version. This one, however, is an internal API, not used by the end user, so it's fine to change this. There are a lot of them in `KeyStateOf` along with new versions of methods since that's a part of public API

Reply:
That makes sense. I'll remove deprecated methods but leave factory methods public in case anyone wants to re-use them while implementing custom instances.

Reply:
Done.